internal/pkg/log/assert.go

```diff
@@ -77,9 +77,17 @@ func valueMatches(value any, actualValue any) bool {
 	return reflect.DeepEqual(actualValue, value)
 }
 
+type tHelper interface {
+	Helper()
+}
+
 // AssertJSONMessages checks that expected json messages appear in actual in the same order.
 // Actual string may have extra messages and the rest may have extra fields. String values are compared using wildcards.
 func AssertJSONMessages(t assert.TestingT, expected string, actual string, msgAndArgs ...any) bool {
+	if h, ok := t.(tHelper); ok {
+		h.Helper()
+	}
+
```

Comment:
:thinking:: ok :heavy_check_mark: 

I would prefer the TB interface, it's better to complicate it in tests than in production code, but ok.